README.md

```diff
@@ -43,7 +43,7 @@ A public list of companies that are known to use the Robot Operating System (ROS
 | [Apptronik](https://apptronik.com/) | Builds and designs human-centered robotics systems. [Draco](https://apptronik.com/our-work/) is Apptronik’s first biped. The system can be a platform for any humanoid system, with easy integration between systems because of the open access to low level control software and libraries for RT Linux/ROS[^120] integration. | 2015 |
 | [ARAV](https://www.arav.jp/) | Offer automated construction equipment. Sponsor of the [ROScon](https://roscon.ros.org/2022/). | 2020 | 
 | [ARC Specialties](http://www.arcspecialties.com) | Design and build automated manufacturing systems and custom equipment for joining, cladding, and other applications. The company is [currently members](https://rosindustrial.org/ric/current-members/#arc-specialties) of ROS-Industrial, is an open-source project that extends the advanced capabilities of ROS to manufacturing automation and robotics (see [`ros-industrial`](https://github.com/ros-industrial))[^131]. | 1983 |
-| [Arendai](https://www.arendai.com/) | Logistics automation enabled by robotics, AI & edge computing. Member of the Intel® Network Builders program. ROS2 Foxy|Galactic with Nav2 are used as an interface to control robots (simulations and own robots). | 2018 | 
+| [Arendai](https://www.arendai.com/) | Seamless integration and interoperability of heterogeneous robotics solutions. Traffic Management and Navigation for optimal orchestration of mobile robots through a single solution using ROS 2. Member of the Intel® Network Builders program. | 2019 | 
```

Comment:
```suggestion
| [Arendai](https://www.arendai.com/) | Seamless integration and interoperability of heterogeneous robotics solutions. Traffic Management and Navigation for optimal orchestration of mobile robots through a single solution using ROS 2. ROS2 Foxy|Galactic with Nav2 are used as an interface to control robots (simulations and own robots). Member of the Intel® Network Builders program. | 2019 | 
```

Reply:
Looking good @ljaniec, only, I'd love to keep the ROS 2 and Nav2 info as it could be relevant for other to know where your expertise lies. Let me know if you're ok with it.

Reply:
Moving forward here @ljaniec, let me know if you'd like any further changes!

Reply:
The mistake is still there:
![image](https://user-images.githubusercontent.com/11331846/193519847-f8e01e44-3cda-47fe-93da-e6fd34fd2595.png)


> ROS2 Foxy|Galactic with Nav2 are used as an interface to control robots (simulations and own robots)

should be 

> ROS2 Foxy & Galactic with Nav2 are used as an interface to control robots (simulations and own robots)

The proposed changes restored the erroneously used "|".

Reply:
Thanks for the hint @ljaniec, my bad. https://github.com/vmayoral/ros-robotics-companies/commit/a47e0a2edc307a014604cb1aebdb409ce79f8410 should do it.

Cheers,